api/server/router/build/build_routes.go

```diff
@@ -92,6 +50,7 @@ func newImageBuildOptions(ctx context.Context, r *http.Request) (*types.ImageBui
 	options.CPUSetCPUs = r.FormValue("cpusetcpus")
 	options.CPUSetMems = r.FormValue("cpusetmems")
 	options.CgroupParent = r.FormValue("cgroupparent")
+	options.Tags = r.Form["t"]
```

Comment:
FormValue returns first value while we want all values of the tag.
 
https://golang.org/pkg/net/http/#Request.FormValue:
"FormValue returns the first value for the named component of the query.
...
To access multiple values of the same key, call ParseForm and then inspect Request.Form directly."

FormValue is called for other keys, which invokes ParseForm internally anyways. One such call is sufficient to have parsed the form and be ready to inspect r.Form directly.